lib/sass/script/parser.rb

```diff
@@ -312,7 +313,12 @@ def unary_#{op}
               start_pos = source_position
 
               @css_variable_warning.warn! if @css_variable_warning
-              node(Tree::UnaryOperation.new(assert_expr(:unary_#{op}), :#{op}), start_pos)
+              e = Tree::UnaryOperation.new(assert_expr(:unary_#{op}), :#{op})
```

Comment:
I don't think triggering based on interpolation is the right approach here—`#{- 5% / 2}` shouldn't parse the same as `#{-5% / 2}`. The lexer should really be lexing `-5%` as a single token.